Client/Frontend/Browser/New Tab Page/NewTabPageViewController.swift

```diff
@@ -169,9 +200,15 @@ class NewTabPageViewController: UIViewController, Themeable {
         }
     }
     
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        checkForUpdatedFeed()
```

Comment:
This check is done in `viewWillAppear` on purpose?
So feed update is called when switching tabs for example

Reply:
Yep! On purpose